cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -23,16 +24,18 @@ import (
 type Commander struct {
 	logger  *zap.Logger
 	cfg     config.Agent
+	logsDir string
 	args    []string
 	cmd     *exec.Cmd
 	doneCh  chan struct{}
 	exitCh  chan struct{}
 	running *atomic.Int64
 }
 
-func NewCommander(logger *zap.Logger, cfg config.Agent, args ...string) (*Commander, error) {
+func NewCommander(logger *zap.Logger, logsDir string, cfg config.Agent, args ...string) (*Commander, error) {
```

Comment:
Should we set the process's current working directory to the persistent storage directory and have everything be relative to that, or should we keep that as a separate option?

Reply:
A separate option is a little redundant. How about using an empty string for `logsDir` to refer to the process's current working directory?

Reply:
Like, the supervisor processes CWD? 

I think we should avoid depending on the global state like that, it can get confusing for users in some cases if they end up using relative paths in configs. I also think if we want to have the supervisor usable as a library, that could be a problem if we are modifying the CWD.

Reply:
Sorry, I mean that we allow setting the Collector process's CWD, and by default in the Supervisor set it to the persistent storage directory. My main thought here is that any relative paths the Collector writes to will be written to a directory we already know at least the Supervisor has R/W access to.

I'm also fine just specifying the log output directory and we can address this separately.

Reply:
Ah, I see. I think we can do that in a follow-up PR, seems like it should probably be an extra configuration option.